tensorflow_model_optimization/python/core/clustering/keras/cluster.py

```diff
@@ -56,6 +56,7 @@ def cluster_scope():
 def cluster_weights(to_cluster,
                     number_of_clusters,
                     cluster_centroids_init,
+                    preserve_sparsity=False,
```

Comment:
I'm wondering, since `perserve_sparsity` is available through experimental.cluster, here `preserve_sparsity` should be fixed inside the function (not available as function arg).

Reply:
That's actually a mistake! In fact we don't have it in our internal master. I've now removed the `preserve_sparsity` from the method's signature.